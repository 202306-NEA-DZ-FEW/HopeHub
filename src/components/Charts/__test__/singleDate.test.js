import renderer from "react-test-renderer";

import SingleDate from "../SingleDate";

it("renders correctly", () => {
    const tree = renderer.create(<SingleDate />).toJSON();
    expect(tree).toMatchSnapshot();
});
