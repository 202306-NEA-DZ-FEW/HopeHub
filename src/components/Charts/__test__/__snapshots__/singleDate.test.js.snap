// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="border border-NeutralBlack w-full col-start-2 col-end-3 row-start-2 h-full overflow-auto"
>
  <table
    className="w-full border corder-black p-6 border-spacing-3"
  >
    <thead>
      <tr
        className="bg-Accent p-0.5 "
      >
        <th
          className="border h-5 w-3/5 border-gray-200 bg-transparent rounded"
        >
          User
        </th>
        <th
          className="border h-5 w-2/5 border-gray-200 bg-transparent rounded"
        >
          Time
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
